src/game.h

```diff
@@ -411,7 +411,7 @@ class game
 
         void open_gate( const int examx, const int examy, const ter_id handle_type );
 
-        bionic_id random_good_bionic() const; // returns a non-faulty, valid bionic
+        std::string random_good_bionic() const; // returns a non-faulty, valid bionic
```

Comment:
Yeah, there are a bunch of functions in `player` as well who's implementation could also be moved to `bionic.cpp`. What do you think?